packages/wonder-blocks-clickable/src/components/__docs__/clickable-behavior.argtypes.js

```diff
@@ -0,0 +1,58 @@
+import clickableArgtypes from "./clickable.argtypes.js";
+
+export default {
+    children: {
+        description:
+            "A function that returns the a React `Element`.  The function is passed an object with three boolean properties: hovered, focused, and pressed, and a `childrenProps` argument that contains all the event handlers that should be passed to the React `Element` itself.",
+        control: {
+            type: "text",
```

Comment:
Is `"text"` usually the type we want for `children`'s control? I think storybook usually default this to a JSON object, and I've been keeping it like that.

Reply:
Good question.. Storybook has this existing issue that is that is not able to parse JSX in controls, so we have to use a different control for that. The issue with `json` is that it's not possible to modify it, and `string` is a bit more configurable in that regard.

Here's an extract from their docs:
> Complex values such as JSX elements cannot be serialized to the manager (e.g., the Controls addon) or synced with the URL. Arg values can be "mapped" from a simple string to a complex type using the mapping property in argTypes to work around this limitation. It works in any arg but makes the most sense when used with the select control type.

Reply:
Giving a second thought, I think that it doesn't matter which option we use, so I'll change it to whatever SB sets as default, that is `json` as you suggested :).